site/sass/components/_card-study.scss

```diff
@@ -0,0 +1,162 @@
+@use 'sass:math';
+
+.card-study {
+  $break-2-col: 980px;
+  $break-feature-col: 690px;
+  $break-featured-stack: 0 689px;
+  $scale-max: 1600px;
+  $clamp-check: clamp(1px, 1vw, 2px);
+
+  border-radius: 0.625rem;
+  display: grid;
+  grid-template-rows: min-content;
+  overflow: hidden;
+  position: relative;
+
+  @include mq($break-2-col) {
+    grid-template: auto / 41.5% 1fr;
+  }
+
+  &,
+  &__tile {
+    background-color: var(--white);
+  }
+
+  &__media {
+    aspect-ratio: 2 / 1;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  &__tile {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: column;
+    padding: 2rem 1.5rem;
+
+    @include mq(0 419px) {
+      align-items: center;
+      text-align: center;
+    }
+
+    @include mq($break-2-col) {
+      padding-bottom: 2rem;
+    }
```

Comment:
```suggestion
```
Looking at the overall styling for this, your bottom padding is already `2rem` here thanks to line 35